ddtrace/internal/symbol_db/remoteconfig.py

```diff
@@ -0,0 +1,47 @@
+from ddtrace.internal.forksafe import has_forked
+from ddtrace.internal.logger import get_logger
+from ddtrace.internal.remoteconfig._connectors import PublisherSubscriberConnector
+from ddtrace.internal.remoteconfig._publishers import RemoteConfigPublisher
+from ddtrace.internal.remoteconfig._pubsub import PubSub
+from ddtrace.internal.remoteconfig._subscribers import RemoteConfigSubscriber
+from ddtrace.internal.remoteconfig.worker import remoteconfig_poller
+from ddtrace.internal.runtime import get_ancestor_runtime_id
+from ddtrace.internal.symbol_db.symbols import SymbolDatabaseUploader
+
+
+log = get_logger(__name__)
+
+
+def _rc_callback(data, test_tracer=None):
+    if get_ancestor_runtime_id() is not None and has_forked():
+        log.debug("Disabling Symbol DB in forked process")
+        # We assume that forking is being used for spawning child worker
+        # processes. Therefore, we avoid uploading the same symbols from each
+        # child process. We restrict the enablement of Symbol DB to just the
+        # parent process and the first fork child.
```

Comment:
@ojung I've added this logic to prevent uploading the same data from multiple children. The risk here is that multiprocessing might be used for other than spawning clones of the same worker process, which means that we might have an incomplete symbol picture in the backend. However, this should be a rare circumstance.